tests/test_scipy.py

```diff
@@ -231,6 +231,12 @@ def test_bad_constraint():
         m.scipy(constraints=[{}])
 
 
+def test_bad_options():
+    m = Minuit(fcn, a=1, b=2)
+    with pytest.raises(ValueError):
+        m.scipy(options=[])
+
+
```

Comment:
Would be nice to also have a test cases that shows the benefit of setting options.